app/models/observation_field_value.rb

```diff
@@ -315,6 +315,7 @@ def as_indexed_json
       name_ci: observation_field.name,
       value: self.value,
       value_ci: self.value,
+      updater_id: updater_id,
```

Comment:
This works, but you should also describe this new field and its type around https://github.com/inaturalist/inaturalist/blob/main/app/es_indices/observation_index.rb#L180. This removes some guesswork on the part of elasticsearch and ensures this field is present if/when the index gets rebuilt. It also documents the entire shape of the schema for other developers.